packages/spectacular-docs/docs/feature-testing/angular-testing-library.md

```diff
@@ -0,0 +1,575 @@
+          featureModule: CrisisCenterModule,
+          featurePath: crisisCenterPath,
+        }),
+      ],
+    });
+    featureRouter = result.debugElement.injector.get(SpectacularFeatureRouter);
+  });
+
+  let featureRouter: SpectacularFeatureRouter;
+  let result: RenderResult<SpectacularAppComponent, SpectacularAppComponent>;
+});
+```
+
+Alternatively, we use
+[`TestBed.inject`](https://angular.io/api/core/testing/TestBed#inject) to
+resolve Spectacular's feature-aware services:
+
+```ts {1,24,27}
+import { TestBed } from '@angular/core/testing';
+import {
+  SpectacularAppComponent,
+  SpectacularFeatureRouter,
+  SpectacularFeatureTestingModule,
+} from '@ngworker/spectacular';
+import { render, RenderResult } from '@testing-library/angular';
+import {
+  CrisisCenterModule,
+  crisisCenterPath,
+} from '@tour-of-heroes/crisis-center';
+
+describe('Tour of Heroes: Crisis center integration tests', () => {
+  beforeEach(async () => {
+    result = await render(SpectacularAppComponent, {
+      excludeComponentDeclaration: true,
+      imports: [
+        SpectacularFeatureTestingModule.withFeature({
+          featureModule: CrisisCenterModule,
+          featurePath: crisisCenterPath,
+        }),
+      ],
+    });
+    featureRouter = TestBed.inject(SpectacularFeatureRouter);
+  });
+
+  let featureRouter: SpectacularFeatureRouter;
+  let result: RenderResult<SpectacularAppComponent, SpectacularAppComponent>;
+});
+```
+
+Either of these techniques can be used to access
+[`SpectacularFeatureLocation`](../api/classes/spectacularfeaturelocation) and
+[`SpectacularFeatureRouter`](../api/classes/spectacularfeaturerouter).
+
+## Navigating to feature routes
+
+When using Angular Testing Library, you can keep using
+[`RenderResult#navigate`](https://testing-library.com/docs/angular-testing-library/api#navigate)
+and your tests will behave as you're used to:
+
+```ts {33}
+import {
+  SpectacularAppComponent,
+  SpectacularFeatureTestingModule,
+} from '@ngworker/spectacular';
+import { render, RenderResult, screen } from '@testing-library/angular';
+import {
+  Crisis,
+  CrisisCenterModule,
+  crisisCenterPath,
+} from '@tour-of-heroes/crisis-center';
+
+describe('Tour of Heroes: Crisis center integration tests', () => {
+  beforeEach(async () => {
+    result = await render(SpectacularAppComponent, {
+      excludeComponentDeclaration: true,
+      imports: [
+        SpectacularFeatureTestingModule.withFeature({
+          featureModule: CrisisCenterModule,
+          featurePath: crisisCenterPath,
+        }),
+      ],
+    });
+  });
+
+  let result: RenderResult<SpectacularAppComponent, SpectacularAppComponent>;
+
+  it('shows crisis detail when a valid ID is in the URL', async () => {
+    const crisis: Crisis = {
+      id: 1,
+      name: 'Dragon Burning Cities',
+    };
+
+    await result.navigate(crisisCenterPath + '/' + crisis.id);
+
+    expect(
+      screen.queryByRole('heading', { name: new RegExp(crisis.name) })
```

Comment:
Since this is the first time that `screen` is used, should we "introduce" it, or add a reference to the docs?